### PR TITLE
Add Chromium real values for Node API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for Chromium for the Node API based upon results from the mdn-bcd-collector project.

### CODE REVIEW

File: api/Node.json [hunk 1]
```diff
@@ -489,10 +489,11 @@
               "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤12.1",
+              "version_removed": "15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": false
```
Comment (foolip) [inline api/Node.json:496]: If `node.getFeature()` was in Opera for desktop then it was almost certainly in Opera for Android too.

File: api/Node.json [hunk 2]
```diff
@@ -636,10 +638,11 @@
               "version_added": "9"
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤12.1",
+              "version_removed": "15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": false
```
Comment (foolip) [inline api/Node.json:645]: Same issue here, `hasAttributes` was probably supported in Presto on Android.

File: api/Node.json [hunk 3]
```diff
@@ -1603,7 +1616,7 @@
               "notes": "Only supported on <code>Element</code>."
             },
             "opera": {
-              "version_added": "7"
+              "version_added": "15"
```
Comment (foolip) [inline api/Node.json:1619]: This can't be right, Presto definitely had `parentElement`. It might have been only on `Element`, perhaps, like IE?

Comment (vinyldarkscratch) [reply api/Node.json:1619]: Ack, this update slipped by -- ah well, I'll add a note and revert the version change!

File: api/Node.json [hunk 4]
```diff
@@ -1603,10 +1618,12 @@
               "notes": "Only supported on <code>Element</code>."
             },
             "opera": {
-              "version_added": "7"
+              "version_added": "7",
+              "notes": "Before Opera 15, this feature was only supported on <code>Element</code>."
```
Comment (foolip) [inline api/Node.json:1622]: I did some quick testing to confirm that `document.body.parentElement` works in Opera 12.16, but `Text` and `Attr` nodes didn't have `parentElement`. Those are the other node types where it most likely might have existing, so I think this was `Element` only.

**Review by foolip — APPROVED** [review]

**Pull request merged**